### ISSUE TITLE
Launch multiple processes of the same instance

### ISSUE THREAD

**Issue by ghost** [PrismLauncher/PrismLauncher#250]
### Role

I use the launcher.

### Suggestion

I want it to be possible to launch the instance multiple times and get multiple clients of the same instance running at the same time.

### Benefit

So you don't have to copy your instance of choice to, for example, use multiple alts at the same time.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

It would need a warning to the user about how it's unsafe to launch the same instance multiple times, just like the vanilla launcher does it. Yes, even the vanilla launcher has that functionality, why can't Prism have it too?

The launcher could also have a system to possibly track multiple log outputs (maybe as multiple collapsible "sections" in the "Minecraft Log" tab).

An easy UI approach to this suggestion would be to leave the launch buttons at the bottom active while the instance is running (by the way, why does the "Launch" button disappear but "Launch Offline" and "Launch Demo" just get grayed out?)

**Comment by MMK21Hub** [comment]
This action could be added to the "File" titlebar menu.

**Comment by ghost** [comment]
> This action could be added to the "File" titlebar menu.

Or by simply not graying out the `Launch` button

![image](https://user-images.githubusercontent.com/110736028/197554617-8442b701-aef9-43a6-a925-bb951872ba8d.png)

**Comment by MMK21Hub** [comment]
My bad, I thought that this issue was referring to opening a new window of the launcher

**Comment by Scrumplex** [comment]
The codebase only expects one process per instance. Apart from the fact that Minecraft probably wouldn't like multiple processes touching the same files either

**Comment by ghost** [comment]
Yeah, but vanilla launcher does allow you to do so (with a warning), why does Prism have to be worse in that specific regard?

**Comment by Bluscream** [comment]
How is this not merged yet? Atleast allow multiple when using shortcuts with different profiles please.

**Comment by Trial97** [comment]
> How is this not merged yet? Atleast allow multiple when using shortcuts with different profiles please.

This is not a problem of merging, no Pull Request fully fixed this issue yet.
As this is an open-source project, if you find it missing a feature, open a Pull Request that implements it yourself.
We are eager to get new contributors.

**Comment by RedHeadEmile** [comment]
I really would like this feature to be implemented too.

**Comment by Kat-09** [comment]
I would also like to see this get implemented.

**Comment by illegalargumentexc** [comment]
I'd also like to see this be implemented

**Comment by Lassebq** [comment]
I'd like to see this implemented. Very useful whenever you want to join a multiplayer server from multiple accounts. Also avoids unnecessary setup for an instance with common/shared data

**Comment by DirectivesMods** [comment]
This needs to be implemented, not sure why or how it isn't a feature. Someone mentioned that no pull request has fully addressed this issue yet, but realistically it should hopefully be easy to implement.

1. Make sure the launch button isn't grayed out
2. Make multiple "tabs" for the minecraft log (for each running instance)

**Comment by maxim0p** [comment]
Requested over 2 years ago and never added, literally just make the launch button clickable.

Crazy how this feature is not present in both Prism and Modrinth launchers, yet somehow the shittiest launcher, Curseforge, is the only launcher that lets you do this.

The most practical use of running multiple Minecrafts on the same instance is to host a LAN world on one of them. This is by far the fastest and most effective way to bug test modpacks in a multiplayer environment, as any bugs in a LAN world typically mean it would happen in a multiplayer server as well. Having to duplicate a profile, and copy over any changes to the other profile, makes it such a pain the ass, that I would rather bug test on Curseforge's launcher.

**Comment by diegogarciarojo** [comment]
You can kill the Prism Launcher process, and you can relaunch minecraft

**Comment by Slingexe** [comment]
Bump, this would be nice instead of having to copy the same instance over and over to run it multiple times

**Comment by netgoatfr** [comment]
bump too, i'd be nice

**Comment by SwordJhon** [comment]
I would also like to see this feature implemented.

**Comment by larroxtv** [comment]
Bump, i need it!

**Comment by Squidgical** [comment]
kinda ridiculous that the only way to do this currently is to mess around with symlinks or some other nonsense.

instead of one process per instance, why not one process of a given instance per account? gives a clear and intuitive point of separation for the user that ought to be quite elegant in code.

alternatively, add a checkbox in the settings labelled something like "shut up I know what I'm doing" that prevents the launch button from ever being disabled?

**Comment by ozymandiu5** [comment]
Bump this. Would be really convenient, and it's already a feature in the vanilla launcher.

**Comment by AnCarsenat** [comment]
Great feature ! I agree.

**Comment by larroxtv** [comment]
Bump.

**Comment by muhammetmatrix** [comment]
Bump

**Comment by WWYDF** [comment]
Bump

**Comment by DSantiagoBC** [comment]
Bump!

**Comment by debuxxed** [comment]
bump

**Comment by Jp8g** [comment]
bump!

**Comment by Ryex** [comment]
Please *STOP* trying to "bump" this, we haven't forgotten.

This issue is much deeper than just allowing another process to run. the internals inherited from multimc in regards to logs and console among other things assume one process per instance and disentangling that is not an easy task.

**Comment by DSantiagoBC** [comment]
Thanks @Ryex ! It's good to know this is still in the radar.

**Comment by HelixCraft** [comment]
**you should definitely add this!**

**Comment by AnCarsenat** [comment]
bump
